### PR TITLE
Added vscode launch to Manage Python Environments tab

### PR DESCRIPTION
**Pull request by anssakthi**
Features Included:
#. Added functionality to launch VSCode to the "Manage Python Environments" tab.
#. Implemented checkboxes in the final page of the installation process. Users can now choose to launch the application and create a shortcut during installation.

### CODE REVIEW

File: doc/source/installer.rst [hunk 1]
```diff
@@ -334,8 +334,10 @@ On the ``Launching options`` section, the following options are available:
 
 * ``Launch Console``: this option starts a console window with the command ``python`` pointing
   towards your selected Python environment.
-* ``Launch JupyterLab``: this option starts a ``JupyterLab`` session. If ``JupyterLab`` is
+* ``Launch VSCode``: this option starts a ``Visual Studio Code``. If ``Visual Studio Code`` is
   not installed, then the ``Ansys Python Manager`` installs it for you.
+* ``Launch JupyterLab``: this option starts a ``JupyterLab`` session. If ``JupyterLab`` is
+  not installed, then the ``Ansys Python Manager`` provides instructions to install it.
```
Comment (RobPasMue) [inline doc/source/installer.rst:340]:
```suggestion
* ``Launch VSCode``: this option starts a ``Visual Studio Code``. If ``Visual Studio Code`` is
  not installed, then the ``Ansys Python Manager`` provides instructions to install it.
* ``Launch JupyterLab``: this option starts a ``JupyterLab`` session. If ``JupyterLab`` is
  not installed, then the ``Ansys Python Manager`` installs it for you.
```

File: src/ansys/tools/installer/uninstall.py [hunk 2]
```diff
@@ -208,6 +208,13 @@ def _remove_all_installed_python(self):
         for path in find_miniforge_linux(ansys_manager_installed_only=True):
             shutil.rmtree(path, ignore_errors=True)
 
+    def _remove_all_installed_python(self):
+        for path in find_ansys_installed_python_linux():
+            path = path.split("bin")[0]
+            shutil.rmtree(path, ignore_errors=True)
+        for path in find_miniforge_linux(ansys_manager_installed_only=True):
+            shutil.rmtree(path, ignore_errors=True)
+
```
Comment (RobPasMue) [inline src/ansys/tools/installer/uninstall.py:217]: Why has this been added and where is it called?

File: setup.nsi [hunk 3]
```diff
@@ -22,9 +22,19 @@ OutFile "dist\${OUTFILE_NAME}"
...
+  CreateShortCut "$desktop\Ansys Python Manager.lnk" "$INSTDIR\Ansys Python Manager.exe"
+FunctionEnd
+
+!define MUI_FINISHPAGE_RUN "$INSTDIR\Ansys Python Manager.exe"
+!define MUI_FINISHPAGE_SHOWREADME
+!define MUI_FINISHPAGE_SHOWREADME_TEXT "Create Desktop Shortcut"
+!define MUI_FINISHPAGE_SHOWREADME_NOTCHECKED
+!define MUI_FINISHPAGE_SHOWREADME_FUNCTION "CreateDesktopShortCut"
+!insertmacro MUI_PAGE_FINISH
+
```
Comment (RobPasMue) [inline setup.nsi:37]: I need to verify the installer works with these changes now.

File: src/ansys/tools/installer/vscode.py [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+"""VS code launch window."""
+import os
+
+from PySide6 import QtCore, QtGui, QtWidgets
+
+from ansys.tools.installer.constants import ANSYS_FAVICON, USER_PATH
+
+
+class VSCode(QtWidgets.QWidget):
+    """VS code launch window."""
```
Comment (RobPasMue) [inline src/ansys/tools/installer/vscode.py:32]: Need to test out locally the behavior of this class now

File: src/ansys/tools/installer/vscode.py [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+        icon = QtGui.QIcon(ANSYS_FAVICON)
+        self.user_confirmation_form.setWindowIcon(icon)
+        self.user_confirmation_form.resize(400, 40)
+        self.user_confirmation_form.setWindowFlag(
+            QtCore.Qt.WindowCloseButtonHint, False
+        )
+        self.user_confirmation_form.show()
+
+    def is_vscode_installed(self):
+        """Check vs code installed or Not.
```
Comment (RobPasMue) [inline src/ansys/tools/installer/vscode.py:174]:
```suggestion
        """Check if VSCode is installed or not.
```

File: src/ansys/tools/installer/vscode.py [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+
+        Returns:
+            Boolean
```
Comment (RobPasMue) [inline src/ansys/tools/installer/vscode.py:177]:
```suggestion
        Returns
        -------
        bool
            Whether VSCode is installed or not.
```

**Review by RobPasMue — CHANGES_REQUESTED** [review]

File: src/ansys/tools/installer/uninstall.py (hunk 2 above)
Comment (anssakthi) [reply src/ansys/tools/installer/uninstall.py:217]: My bad, the function has been declared already.

**Review by RobPasMue — APPROVED** [review]
LGTM! Feel free to merge @anssakthi - thanks for the implementation

**Pull request merged**